packages/components/input-number/src/input-number.vue

```diff
@@ -17,10 +17,12 @@
       :class="[ns.e('decrease'), ns.is('disabled', minDisabled)]"
       @keydown.enter="decrease"
     >
-      <el-icon>
-        <arrow-down v-if="controlsAtRight" />
-        <minus v-else />
-      </el-icon>
+      <slot name="decreaseIcon">
```

Comment:
```suggestion
      <slot name="decrease-icon">
```
Use hyphens to be consistent with other components.

Reply:
Thanks to Vue's handling, this does not affect the results

Reply:
Yes. It’s just that I think it might be better to keep the component styles consistent.

Reply:
okay